qa/tasks/cephfs/test_damage.py

```diff
@@ -454,9 +454,7 @@ def test_damaged_dentry(self):
         scrub_json = self.fs.rank_tell(["scrub", "start", "/subdir", "repair"])
         log.info(json.dumps(scrub_json, indent=2))
 
-        self.assertEqual(scrub_json["passed_validation"], False)
-        self.assertEqual(scrub_json["raw_stats"]["checked"], True)
-        self.assertEqual(scrub_json["raw_stats"]["passed"], False)
+        self.assertEqual(scrub_json["return_code"], 0)
```

Comment:
This used to be a synchronous call to scrub. All scrubs are now asynchronous so we actually need to wait for it to complete. That involves running `scrub status` until the scrub no longer shows up. Can you code up that poll method and add it to `filesystem.py`. It seems to be a common thing we're doing (see also `qa/tasks/fwd_scrub.py`).

Reply:
Yeah, sure. will do that.